src/main/java/com/nametagedit/plugin/web/EditorSession.java

```diff
@@ -0,0 +1,184 @@
+package com.nametagedit.plugin.web;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.nametagedit.plugin.NametagEdit;
+import com.nametagedit.plugin.NametagHandler;
+import com.nametagedit.plugin.api.data.GroupData;
+import com.nametagedit.plugin.api.data.PlayerData;
+import com.nametagedit.plugin.storage.AbstractConfig;
+import com.nametagedit.plugin.web.json.JsonConverter;
+import com.nametagedit.plugin.web.json.JsonTrack;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+@RequiredArgsConstructor
+@Getter
+public class EditorSession {
+    private static final String EDITOR = "http://localhost:3000/";
+    private static final URL PASTE;
+    private final NametagEdit plugin;
+    private final CommandSender player;
+    @Setter
+    @Nullable private String code;
+
+    static {
+        URL paste = null;
+
+        // todo edit
```

Comment:
load from config file ?

Reply:
This is a work in progress, when the web editor will be ready I’ll change it